SU-SWS/stanford_person
# VPSADEV-242: Move the content from content to full width region on the huge view mode

# how to test

1. Checkout this branch
2. Revert the stanford_person feature
3. Clear caches
4. Create a person node and set view mode on display setting to 'huge'
5. Review layout
6. Edit node and select default or full view mode
7. See the view mode and content position change.

It looks like a few title changes got in there as well. I exported this module from the development site so I'm not sure if that is the intended language or not. Please have a review below.
